Class-4-(Classes-Part-2).md

```diff
@@ -0,0 +1,37 @@
+# Class 4: Classes, Part 2
+
+## In the prep work this week, the students learned
+
+1. What a class is in Java and how instantiating classes create objects.
+1. The difference between fields and methods
+1. What access modifiers are and how they can be used to encapsulate data.
+1. What the Single Responsibility Principle is and why we should make our classes responsible for one thing.
+
```

Comment:
These are the same as Lesson 3!

Reply:
Oops.  I was a little to cavalier about making sure I changed ALL of the sections in the cut-and-paste template from class 3.

Fixed.